sdk/FEATURES.md

```diff
@@ -234,6 +235,12 @@ It also includes several annontations, `tekton.dev/input_artifacts` and `tekton.
 The current implementation is relying on the existing KFP's minio setup for getting the default credentials. These default credentials can be updated
 via a Kubernetes configmap.
 
+### Caching
+
+By default compiling a pipeline will add metadata annotations and labels so that results from tasks within a pipeline run can be re-used if that task is reused in a new pipeline run. This saves the pipeline run from re-executing the task when the results are already known. 
+
+The specific annotations and labels that are added to the task spec metadata to enable caching are: `annotations={'tekton.dev/template': ""}` and `labels={'pipelines.kubeflow.org/cache_enabled': 'true', 'pipelines.kubeflow.org/pipelinename': '', 'pipelines.kubeflow.org/generation': ''}`.
+
```

Comment:
Can you tell the users that cache is enabled by default. If they want to disable cache, they need to use the label from your example.
```
task2.add_pod_label('pipelines.kubeflow.org/cache_enabled', 'false')
```